src/main/java/seedu/address/ui/CountryNoteListPanel.java

```diff
@@ -1,36 +1,92 @@
 package seedu.address.ui;
 
+import javafx.collections.ListChangeListener;
 import javafx.collections.ObservableList;
 import javafx.fxml.FXML;
+import javafx.scene.Node;
 import javafx.scene.control.Label;
-import javafx.scene.control.ListCell;
-import javafx.scene.control.ListView;
+import javafx.scene.control.ScrollPane;
 import javafx.scene.layout.Region;
 import javafx.scene.layout.VBox;
 import seedu.address.model.country.Country;
 import seedu.address.model.note.CountryNote;
 
 /**
- * A widget that wraps around ListView.
+ * Panel containing the list of country notes.
  */
 public class CountryNoteListPanel extends UiPart<Region> {
-    private static final String FXML = "WidgetListViewBox.fxml";
 
-    @FXML
-    private VBox viewBox;
+    private static final String FXML = "CountryNoteListPanel.fxml";
+    private static final String HEADER_ALL_COUNTRIES_TEXT = "All Country Notes";
+
+    private boolean displayAllCountries = true;
+    private ObservableList<CountryNote> countryNoteObservableList;
+
     @FXML
     private Label header;
     @FXML
-    private ListView<CountryNote> countryNoteListView;
+    private VBox countryNoteListView;
```

Comment:
Should this be `countryListView` since the subpanel contains a `countryNoteListView`?

Reply:
This one is being used to display country notes if we're looking at one country only, so I just left it as countryNoteListView